README.rst

```diff
@@ -31,88 +34,82 @@ Bifravst
    :alt: ESLint: TypeScript
    :target: https://github.com/typescript-eslint/typescript-eslint
    
-.. admonition:: 🐣 incubating *Bifravst* as an official *Nordic Semiconductor* project 🎉
-
-  In the coming weeks, we will establish the best path to provide this project
-  through nRF Connect SDK, Nordic Semiconductor Info center and potentially
-  other established channels. For more information see the
-  `GitHub issue <https://github.com/bifravst/bifravst/issues/56>`_.
 
 Vision
-================================================================================
+******
 
-*Bifravst* aims to provide a concrete end-to-end example for an ultra-low power
-cellular IoT product in the asset tracker space, namely a *Cat Tracker*.
+*Bifravst* aims to provide a concrete end-to-end example for an ultra-low power cellular IoT product in the asset tracker space, namely a *Cat Tracker*.
 
 .. figure:: ./docs/logo-with-text.png
    :alt: Bifravst: Cat Tracker IoT example
 
-.. epigraph::
 
-  *Bifravst* enables the developers to set up a real world IoT solution using
-  the respective cloud provider and adapt the example firmware and software
-  quickly for a specific use case.
+*Bifravst* enables the developers to set up a real world IoT solution using the respective cloud provider and adapt the example firmware and software quickly for a specific use case.
 
 *Bifravst* aims to provide answers and recommend best practices to the following
 questions :
 
-- *How can you connect Nordic's cellular IoT chips to your cloud provider?*
-- *How do devices send data into the cloud?*
-- *How can the data be sent to the devices?*
-- *How can users and other services interact with the devices?*
-- *How can you upgrade the application firmware of your devices while they are
-  deployed in the field?*
-- *How can you develop a cellular IoT product that maximizes battery life,
-  minimizes data usage, and handles unreliable connectivity gracefully?*
+* How can you connect Nordic's cellular IoT chips to your cloud provider?
+* How do devices send data into the cloud?*
+* How can the data be sent to the devices?
+* How can users and other services interact with the devices?
+* How can you upgrade the application firmware of your devices while they are deployed in the field?
+* How can you develop a cellular IoT product that maximizes battery life, minimizes data usage, and handles unreliable connectivity gracefully?
 
-*Bifravst* is licensed under the `3-clause BSD license <./LICENSE>`_ (software)
```

Comment:
What is the reason for remove the license section?

Reply:
I shall fix this. I tried to move this sentence, but it might have accidentally got deleted. Thanks for spotting this!